applications/minotari_node/Cargo.toml

```diff
@@ -52,7 +52,7 @@ tonic = { version = "0.6.2", features = ["tls", "tls-roots" ] }
 tari_metrics = { path = "../../infrastructure/metrics", optional = true, features = ["server"] }
 
 [features]
-default = ["metrics"]
+default = ["metrics", "libtor"]
```

Comment:
this will break windows builds

Reply:
This is what we're testing.
Including `tari_libtor` shouldn't cause any kind of error, and then inside `tari_libtor`, libtor is actually compiled out on windows. So in theory it shouldn't break because it never really gets included. We also specify not to include tari_libtor for our windows builds. So let's see what happens! 

Don't worry we're not merging until we validate.

Reply:
nice, I don't actually care about windows - but I sweated a lot trying to get it to work on there :weary:  

(the tor project itself doesn't even compile natively on windows, they cross compile from linux with msys2/mingw)

Reply:
oh that's definitely interesting, good point.